addons/website_blog/static/src/snippets/s_blog_posts/000.scss

```diff
@@ -1,296 +1,221 @@
+            .s_blog_posts_post_cover_header, .s_blog_posts_post_cover_hidden {
+                height: 50%;
             }
-            figcaption {
-                &::before {
-                    content: "";
-                    @include o-position-absolute(0, 0, 0, 0);
-                    background: linear-gradient(to bottom, darken(map-get($theme-colors, 'secondary'), 10%) 0%, darken(map-get($theme-colors, 'secondary'), 30%) 100%);
-                    z-index: -1;
-                }
-                padding: 3em;
-                text-align: left;
-                &:after {
-                    @include o-position-absolute(10px, 10px, 10px, 10px);
-                    border: 2px solid #fff;
-                    border-top-width: 4px;
-                    border-bottom-width: 4px;
-                    content: "";
-                    transition: transform-origin 0.35s;
-                    transform: scaleY(0.5);
-                    transform-origin: top;
-                }
+
+            .s_blog_post_big_picture_title, .s_blog_posts_post_cover_hidden {
+                transform: translateY(50%);
+                transition: $transition-base;
             }
-            .s_blog_posts_post_subtitle {
-                @include o-position-absolute(auto, 20px, 20px, 20px);
+
+            .s_blog_post_big_picture_infos::before {
+                @include o-position-absolute(0, 0, 0, 0);
+                background-color: rgba($black, .3);
+                transition: $transition-fade;
+                opacity: 0;
+                content: "";
+            }
+
+            .s_blog_posts_post_cover_hidden, .s_blog_post_big_picture_infos::before {
                 opacity: 0;
-                transition: opacity 0.35s linear, transform 0.35s;
-                transform: translate3d(0,-100px,0);
             }
-            .s_blog_posts_post_title {
-                @include o-position-absolute(20px, 20px, auto, 20px);
+
+            .s_blog_posts_post:hover {
+                .s_blog_posts_post_cover_hidden, .s_blog_post_big_picture_infos:before {
+                    opacity: 1;
+                }
             }
-            &:hover {
-                .o_record_cover_container {
-                    opacity: 0.4 !important;
+
+            // "Bind" base animations.
+            &.o_picture_size_default {
+                @include media-breakpoint-down(lg) {
+                    @include -big-picture-animate-extended();
                 }
-                figcaption::after {
-                    transform-origin: bottom;
+                @include media-breakpoint-only(lg) {
+                    @include -big-picture-animate-compact();
                 }
-                .s_blog_posts_post_subtitle {
-                    opacity: 1;
-                    transform: translate3d(0, 0, 0);
+                @include media-breakpoint-up(xl) {
+                    @include -big-picture-animate-extended();
```

Comment:
Yep, luckily there are not rules in `-big-picture-animate-extended()` that are **not** overridden  by `-big-picture-animate-compact()` when on `lg`, good catch 👍

Reply:
Oh I thought it would be the same in that case as well in fact. But apparently `@include media-breakpoint-down(lg) {` does not include LG so what you coded makes sense. They changed that in BS5 actually, I forgot. I don't find that "more logical" at all:
![image](https://github.com/odoo/odoo/assets/10338094/4e52c416-1fdf-4767-ae99-546c2ca7d1f2)

But indeed, if you override all properties, best goes with what I said to have less CSS rules.